views/build/grunt/bundle.js

```diff
@@ -36,7 +36,9 @@ module.exports = function(grunt) {
                         name : 'qtiPreviewer',
                         babel : true,
                         include : [
-                            'taoQtiTestPreviewer/previewer/**/*'
+                            'taoQtiTestPreviewer/previewer/**/*',
+                            'taoQtiTestPreviewer/previewer/adapter/test/qtiTest',
+                            'taoQtiTestPreviewer/previewer/component/test/qtiTest'
```

Comment:
Is it needed? the line above contains wildcards, so it should not be needed to explicitly mention the sub-folders, unless there is a special trick due to the presence of the `/test/` folder?

Reply:
yes, bundler doesn't include these 2 files because `/test/ `in the path

Reply:
ok, fine!